### PR TITLE
flat-remix-icon-theme: init at 20191018

### PR DESCRIPTION
**Pull request by feijoas**
###### Motivation for this change

Added the flat-remix-icon-theme

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/data/icons/flat-remix-icon-theme/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, gtk3 }:
+
+stdenv.mkDerivation rec {
+  name = "${package-name}-${version}";
+  package-name = "flat-remix-icon-theme";
```
Comment (Infinisil) [inline pkgs/data/icons/flat-remix-icon-theme/default.nix]: Use `pname` instead, then you don't need to set the name either and you can remove the `rec` too.

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -3192,6 +3192,10 @@
     email = "matthew@wellquite.org";
     name = "Matthew Sackman";
   };
+  mschneider = {
+    email = "markus.schneider.sic+nix@gmail.com";
+    name = "Markus Schneider";
```
Comment (Infinisil) [inline maintainers/maintainer-list.nix]: Add yourself as a maintainer in a separate commit in this PR

**Review by infinisil — CHANGES_REQUESTED** [review]

File: pkgs/data/icons/flat-remix-icon-theme/default.nix (hunk 1 above)
Comment (feijoas) [reply pkgs/data/icons/flat-remix-icon-theme/default.nix]: I made the changes. Thanks

File: maintainers/maintainer-list.nix (hunk 2 above)
Comment (feijoas) [reply maintainers/maintainer-list.nix]: Done

**Comment by Infinisil** [comment]
The commits should be the other way around though, how it is now, you reference `maintainers.mschneider` before it actually exists

**Comment by feijoas** [comment]
Changed the order of commits.

**Comment by Infinisil** [comment]
@GrahamcOfBorg build flat-remix-icon-theme

**Comment by feijoas** [comment]
Why is the hash on darwin different?

**Comment by c0bw3b** [comment]
@GrahamcOfBorg build flat-remix-icon-theme

File: pkgs/data/icons/flat-remix-icon-theme/default.nix [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    for theme in $out/share/icons/*; do
+      gtk-update-icon-cache $theme
+    done
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Flat remix is a pretty simple icon theme inspired on material design";
+    homepage = https://drasite.com/flat-remix;
+    license = with licenses; [ gpl3 ];
+    platforms = platforms.all;
```
Comment (c0bw3b) [inline pkgs/data/icons/flat-remix-icon-theme/default.nix]:
```suggestion
    platforms = platforms.linux;
```
Let's avoid the filesystem normalisation problem causing the hash mismatch on Darwin.
Plus upstream advertise this as a Linux icon theme anyway :)

**Review by c0bw3b — APPROVED** [review]

**Pull request merged**